checker/tests/nullness/Issue880.java

```diff
@@ -0,0 +1,22 @@
+// Test case for Issue 880:
```

Comment:
The jtreg test uses separate files, whereas this tests within one file. So it covers an additional case.